lib/p2p/Pool.ts

```diff
@@ -728,7 +728,11 @@ class Pool extends EventEmitter {
     const orderInvalidationPacket = new packets.OrderInvalidationPacket({ id, pairId, quantity });
     this.peers.forEach(async (peer) => {
       if (!nodeToExclude || peer.nodePubKey !== nodeToExclude) {
-        await peer.sendPacket(orderInvalidationPacket);
+        if (peer.advertisedPairs.includes(pairId)) {
```

Comment:
Better to use `peer.isPairActive(pairId)` here as sometimes advertised pairs are not active (and therefore we wouldn't be sending orders for that pair to the peer in the first place).